index.html

```diff
@@ -27,6 +27,139 @@
                 <li><a href="#"><img class="social-ico" src="res/twitter.svg" alt="twitter"></a></li>
                 <li><a href="#"><img class="social-ico" src="res/m.svg" alt="meta"></a></li>
             </ul>
-        </section>    
+        </section>
+        <section class="works-section">
+            <div class="my-recent-work">
+                <h1>My Recent Works</h1>
+                <hr class="recent-hr">
+            </div>
+            <div class="works">
+                <div class="project-1">
+                    <img class="project-1-img" src="res/img_Placeholder.png">
+                    <h3>Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="siklls">
+                        <a>css</a>
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
```

Comment:
- [ ] Kindly make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") use a `ul` tag as a container and each of them a `li` tag for best semantics tags and as required for the project
**Make sure you apply this approach on all cards**